libs/community/langchain_community/vectorstores/cratedb/extended.py

```diff
@@ -0,0 +1,92 @@
+import logging
+from typing import (
+    Any,
+    Callable,
+    Dict,
+    List,
+    Optional,
+)
+
+import sqlalchemy
+
+from langchain.schema.embeddings import Embeddings
+
+from langchain_community.vectorstores.cratedb.base import (
+    DEFAULT_DISTANCE_STRATEGY,
+    CrateDBVectorSearch,
+    DistanceStrategy,
+)
+from langchain_community.vectorstores.pgvector import _LANGCHAIN_DEFAULT_COLLECTION_NAME
+
+
+class CrateDBVectorSearchMultiCollection(CrateDBVectorSearch):
+    """
+    Provide functionality for searching multiple collections.
+    It can not be used for indexing documents.
+
+    To use it, you should have the ``crate[sqlalchemy]`` Python package installed.
```

Comment:
That installation incantation will be deprecated. Something like this would be better:

```suggestion
    To use it, you should have the ``sqlalchemy-cratedb`` Python package installed.
```